### DISCUSSION TITLE
VS freezes frequently after installing this component

### DISCUSSION THREAD

**Discussion by franky1017** [Blazored/Modal#304] [Q&A]
My Visual Studio freezes frequently after file saves once I installed this component. I am not sure what's causing it. As soon as I save any file, VS freezes about 30 seconds. Is there any thing special this component is doing that could cause this issue?

**Comment by larsk2009** [comment]
I can't think of anything that could cause this issue. The component us only using standard blazor things, so I don't think it is the cause. I am not very familiar with Visual Studio logging, but I am sure there is some way to figure out what is actually taking so much time from one of the logs or something.
